stacker/stack.py

```diff
@@ -71,14 +71,14 @@ def __init__(self, definition, context, variables=None, mappings=None,
         self.enabled = enabled
         self.protected = protected
         self.context = copy.deepcopy(context)
+        self.outputs = None
 
     def __repr__(self):
         return self.fqn
 
     @property
     def requires(self):
-        requires = set([self.context.get_fqn(r) for r in
-                        self.definition.requires or []])
+        requires = set(self.definition.requires or [])
```

Comment:
Technically this is a breaking change, but one that I doubt would effect very many people. `requires` would no longer support specifying an `fqn`, instead it would only support using the node name in the graph (same as how the `output` lookup works).

Reply:
Can you run this by the folks in the stacker channel, just to make sure we aren't breaking anything for anyone we know of?